### ISSUE TITLE
Case studies/user logos

### ISSUE THREAD

**Issue by odow** [jump-dev/jump-dev.github.io#110]
I've been talking to a lot of JuMP users recently. I think we could do a lot better with advertising the people that are happily using JuMP. Making a list of people that I've talked to recently

## Academia

 *  DTU Denmark (Thomas Stidsen, et al)
 * University of Setif Algeria (Bouras Ikram)
 * Ecole des Ponts
 * Mines Paris
 * Nantes
 * President of ROADEF (https://homepages.laas.fr/sungueve/)
 * TUDelft (https://www.tudelft.nl/en/eemcs/the-faculty/departments/software-technology/algorithmics/projects/nextgenopt)
 * UW Madison

## Government

 * LANL
 * NREL (Sienna, Reopt)
 * ANL
 * SINTEF
 * NZ EA (https://www.emi.ea.govt.nz/Wholesale/Tools/JADE)

## Industry

 * AtOptima
 * Renault (https://github.com/LouisBouvier/InventoryRoutingLNS.jl, https://arxiv.org/abs/2209.00412)

**Comment by sstroemer** [comment]
My small input for that:
- `NextGenOpt` is not ours, as far as I know it's lead by Mathijs de Weerdt (TU Delft), but I don't know them (my affiliation with TU Delft is with the TPM faculty).
- "Our" model will soon (TM) be open-source under "IESopt", affiliation for that would be "AIT Austrian Institute of Technology", which could be similar to SINTEF, LANL, ... in the government section (we are > 50% government owned).
- I know a few "outdated" models ([CapacityExpansion.jl](https://github.com/YoungFaithful/CapacityExpansion.jl), [NemoMod.jl](https://github.com/sei-international/NemoMod.jl), [POMATO](https://github.com/richard-weinhold/pomato))

**Comment by odow** [comment]
> as far as I know it's lead by Mathijs de Weerdt

Yeah I've been talking to some people involved. @abelsiqueira is also part of the team.

**Comment by sstroemer** [comment]
Sorry, one more that I remembered: [PowerModelsACDC.jl](https://github.com/Electa-Git/PowerModelsACDC.jl) from KU Leuven

**Comment by joaquimg** [comment]
I like the logos idea.
Quotes from these people and more could be nice as well.

**Comment by odow** [comment]
JuMP (or Julia) was used by at least three teams in https://gocompetition.energy.gov
ARPA-e Benchmark
TIM-GO 
quasiGrad

**Comment by abelsiqueira** [comment]
@datejada, @clizbe, you can add the model (and TNO's name?) here, if you're up for it.

**Comment by matbesancon** [comment]
In academia: Zuse Institute Berlin, TU Berlin

**Comment by datejada** [comment]
In Energy System Models: TNO and the eScience Center are developing TulipaEnergyModel.jl https://github.com/TulipaEnergy/TulipaEnergyModel.jl

**Comment by matbesancon** [comment]
@stelmo @exaexa can we mention HH Düsseldorf and U. Luxembourg?

**Comment by stelmo** [comment]
Yes! At the Heinrich Heine University in Düsseldorf (me) and University of Luxembourg (@exaexa) we are happily using JuMP to build constraint-based reconstruction and analysis software (e.g. https://github.com/LCSB-BioCore/COBREXA.jl).

**Comment by stelmo** [comment]
In fact, @exaexa recently built an entire framework to make LP/QP modeling simpler to connect to JuMP and all of its solvers: https://discourse.julialang.org/t/ann-constrainttrees-jl-tidy-constrained-systems-for-jump/105563

**Comment by exaexa** [comment]
Hi all, confirming that we use a lot of JuMP too :D

- In academia: [LCSB](https://www.uni.lu/lcsb-en/) at [Uni.Lu](https://www.uni.lu/). Please take logos from https://github.com/COBREXA/ConstraintTrees.jl/tree/master/docs/src/assets . Links: https://github.com/lcsb-biocore/COBREXA.jl/ and https://github.com/LCSB-BioCore/FBCModelTests.jl
- What would be the way to report the [PerMedCoE EU H2020 project](https://permedcoe.eu/) here? Technically that is the framework that enabled a major part of the above developments, and will continue using JuMP in European HPCs. (Just in case, the logo is in the same folder as above)

Thanks!

**Comment by clizbe** [comment]
Clarification: [TulipaEnergyModel.jl](https://github.com/TulipaEnergy/TulipaEnergyModel.jl) is the model in the NextGenOpt project, involving TNO, eScience, TUDelft (Mathijs de Weerdt), and Utrecht University.

**Comment by odow** [comment]
@clizbe oops, sorry. Fixed here: https://github.com/jump-dev/jump-dev.github.io/pull/121

**Comment by clizbe** [comment]
Oh no! That's not what I meant. It's indeed TNO's model - but just clarifying that "the NextGenOpt model" that people were trying to remember is not a separate model, it's Tulipa.
